keybase/platform_windows.go

```diff
@@ -125,6 +139,37 @@ func CheckRegistryUninstallCode(productID string, log Log) bool {
 	return false
 }
 
+// Read all the runonce subkeys and find the one with "Keybase" in the name.
+func checkRebootPending(wow64 bool) (result bool) {
+	var access uint32 = registry.ENUMERATE_SUB_KEYS | registry.QUERY_VALUE
+	if wow64 {
+		access = access | registry.WOW64_32KEY
+	}
+
+	k, err := registry.OpenKey(registry.CURRENT_USER, "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\RunOnce", access)
+	if err != nil {
+		fmt.Printf("Error %s opening RunOnce subkeys\n", err.Error())
+		return
+	}
+	defer k.Close()
+
+	params, err := k.ReadValueNames(0)
+	if err != nil {
+		fmt.Printf("Can't ReadSubKeyNames %#v", err)
+		return
+	}
+
+	for _, param := range params {
+		val, _, err := k.GetStringValue(param)
+
+		if err == nil && strings.Contains(val, "Keybase") {
```

Comment:
I would change this too:

```
if err != nil {
  return false, err
}
if strings.Contains(val, "Keybase") {
  return true, nil
}
```

The `if err != nil` pattern is very common and so is clearer... Also short circuiting early is better than more complex if statements.

(yeah this is a bit nitpickey.)